engines/cge2/vmenu.cpp

```diff
@@ -155,7 +155,8 @@ void VMenu::touch(uint16 mask, V2D pos, Common::KeyCode keyCode) {
 
 		Common::TextToSpeechManager *ttsMan = g_system->getTextToSpeechManager();
 		if (_lastN != n) {
-			ttsMan->say(_menu[n]->_text, Common::TextToSpeechManager::INTERRUPT);
+			if (ttsMan != nullptr && ConfMan.getBool("tts_enabled_objects"))
```

Comment:
You also added a check without explaining why.

Reply:
The game has two options for TTS:
1. ConfMan.getBool("tts_enabled_objects") for options and object descriptions
2. ConfMan.getBool("tts_enabled_speech") for subtitles

I think the quit dialog fits more into the first category.

Reply:
I was a little too terse.
I meant you should add a sentence in your commit message to explain that you added a missing check for settings.
Thanks

Reply:
Updated. Hope it's not too verbose for a one line change.